wsmaster/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth1/OAuthAuthenticationException.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.security.oauth1;
+
+/**
+ * Exception raised when the OAuth authentication failed.
+ *
+ * @author Kevin Pollet
+ */
+public final class OAuthAuthenticationException extends Exception {
```

Comment:
It can be thrown by Service. Would it better to extend this exception from `ServerException` or maybe from another ApiException?